packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -211,47 +212,68 @@ const prompts = () => {
         s === '' ||
         defaultEinsteinAPIError
 
+    const demoChoice = 'Yes'
+    const customChoice = 'No, use custom settings'
+
     const questions = [
+        {
+            name: 'useDemoSettings',
+            message: 'Do·you·want·to·try·out·PWA·Kit·with·the·demo·storefront?',
+            choices: [demoChoice, customChoice],
+            type: 'list'
+        },
         {
             name: 'projectId',
             validate: validProjectId,
-            message: 'What is your project ID (example-project) in Managed Runtime Admin?'
+            message: 'What is your project ID (example-project) in Managed Runtime Admin?',
+            when: (answers) => answers.useDemoSettings == customChoice
```

Comment:
How about instead of repeating this function on every question, we do something like this?

```js
    const customChoiceQuestions = [
        {
            name: 'projectId',
            validate: validProjectId,
            message: 'What is your project ID (example-project) in Managed Runtime Admin?'
        },
        // and the rest of the questions
    ]

    const questions = [
        {
            name: 'useDemoSettings',
            message: 'Do you want to try out PWA Kit with the demo storefront?',
            choices: [demoChoice, customChoice],
            type: 'list'
        },
        ...customChoiceQuestions.map((question) => ({
            ...question,
            when: (answers) => answers.useDemoSettings == customChoice
        }))
    ]
```

Reply:
Good suggestion. I'll make this change

Reply:
This change has been applied and tested @vmarta